honeybee_vtk/assistant.py

```diff
@@ -37,6 +37,16 @@ def __init__(self, background_color: Tuple[int, int, int], camera: Camera,
         self._camera = camera
         self._legend_params = legend_parameters
         self._text_actor = text_actor
+        self._vtk_camera = None
+        self._active_camera = None
```

Comment:
Why are you adding `_active_camera` here if you never using it in the class?

Reply:
It's a remnant of an old design I was working on. I removed it.